src/tabs/Tabs.tsx

```diff
@@ -119,11 +122,39 @@ export class Tabs extends PureComponent<TabsProps, TabsState> {
     }
   }
 
+  moveHighlight = (activeTabId: string) => {
+    const currentTab = this.state.tabIdToRefs.get(activeTabId).current
+    const tabBounds = currentTab.getBoundingClientRect()
+
+    // tablist-wrapper parent needed to get the height of the whole component
+    // because tabs can have different heights
+    const parentTabWrapper: any = currentTab.closest('.kirk-tablist-wrapper')
```

Comment:
`any` because if I type it as what it is (HTMLDivElement) TypeScript won't let me commit:
```
error TS2740: Type 'Element' is missing the following properties from type 'HTMLDivElement': align, accessKey, accessKeyLabel, autocapitalize, and 110 more.

131     const parentTabWrapper: HTMLDivElement = currentTab.closest('.kirk-tablist-wrapper')
```

If I don't provide type, my linter complains that I can't use `offsetHeight` from that element...

Reply:
Use HTMLElement instead of HTMLDivElement, that should work.
This is static type-checking, the fact that it is a div in particular is runtime and can't be enforced at compile time.

Reply:
Same thing with HTMElement

Reply:
Alright, left a comment in the code about why this happens.
".closest" returns a base Element that doesn't have offsetHeight so we cast it to HTMLElement using the "as" TS syntax